src/Phansible/Resources/ansible/roles/beanstalkd/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Discover if there is already a version of beanstalkd installed.
+  shell: which beanstalkd > /dev/null && beanstalkd -v | cut -d' ' -f2
+  changed_when: False
+  failed_when: False
+  register: installed_beanstalkd_version
```

Comment:
What this holds when the first command fails? And how that affects the rest of the playbook?